db/seeds/branches_of_service.yml

```diff
@@ -0,0 +1,150 @@
+- name: Army
+  display_order: 1
+  contact:
+    claims_dsn: 536-3000
+    claims_tel_comm: (502) 626-3000
+    claims_email: usarmy.knox.hqda-otjag.mbx.cpcs@mail.mil
+  posts:
+    - title: Temporary Change of Station (TCS) Guidance
+      effective_at: 20170525
+      content: |
+        <p>If you have a Temporary Change of Station (TCS) coming up, it is recommended that you visit your local property office for help arranging your move. They will be able to help you schedule your move and deal with the more complicated entitlements and accounting procedures than are typically associated with your normal Permanent Change of Station (PCS).</p>
+        <a download="tcs-guidance.docx" href="/documents/army-tcs-guidance.docx">View Full Temporary Change of Station (TCS Guidance)</a>
```

Comment:
I put these documents in /public/documents instead of assets because otherwise I couldn't reference them from here